src/Sulu/Bundle/AdminBundle/Resources/js/views/ResourceTabs/ResourceTabs.js

```diff
@@ -77,3 +78,10 @@ export default class ResourceTabs extends React.Component<ViewProps> {
         );
     }
 }
+
+export default withSidebar(ResourceTabs, function() {
```

Comment:
I don't think that this is the right place... That should not appear on all forms... Can't we add that only to the page form?

Reply:
this will be improved in the next PR (preview implementation)

Reply:
Well, who knows when the next PR will get merged, and this sidebar doesn't really look nice :see_no_evil: Can we agree that we remove that part after we have fully tested it? So that we can merge it without an actual appearing sidebar?